### PR TITLE
Update links in README.md to point to Element

### PR DESCRIPTION
**Pull request by Weasy666**
There still one or two links pointing to Riot specific stuff, as there exists no equivalent for Element, at least as far as i know.

These are:
- [https://packages.riot.im/riot-release-key.asc](https://github.com/Weasy666/element-web/blame/develop/README.md#L49)
- [https://matrix.to/#/#riot-dev:matrix.org](https://github.com/Weasy666/element-web/blame/develop/README.md#L195)
- [https://translate.riot.im/engage/element-web/?utm_source=widget](https://github.com/Weasy666/element-web/blame/develop/README.md#L339)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,9 +37,9 @@ for those who like living dangerously.
 To host your own copy of Element, the quickest bet is to use a pre-built
 released version of Element:
 
-1. Download the latest version from https://github.com/vector-im/riot-web/releases
+1. Download the latest version from https://github.com/vector-im/element-web/releases
 1. Untar the tarball on your web server
-1. Move (or symlink) the `riot-x.x.x` directory to an appropriate name
+1. Move (or symlink) the `element-x.x.x` directory to an appropriate name
```
Comment (jryans) [inline README.md:42]: This has not been changed yet AFAIK, see https://github.com/vector-im/element-web/issues/14896.

File: README.md [hunk 2]
```diff
@@ -126,31 +126,31 @@ override the desktop app's default settings if desired.
 Running from Docker
 ===================
 
-The Docker image can be used to serve riot-web as a web server. The easiest way to use
+The Docker image can be used to serve element-web as a web server. The easiest way to use
 it is to use the prebuilt image:
 ```bash
-docker run -p 80:80 vectorim/riot-web
+docker run -p 80:80 vectorim/element-web
```
Comment (jryans) [inline README.md:132]: This has not been done yet, see https://github.com/vector-im/element-web/issues/14930.

File: README.md [hunk 3]
```diff
@@ -186,12 +186,12 @@ Element requires the following URLs not to be cached, when/if you are serving El
...
 
-Before attempting to develop on Riot you **must** read the [developer guide
+Before attempting to develop on Element you **must** read the [developer guide
 for `matrix-react-sdk`](https://github.com/matrix-org/matrix-react-sdk#developer-guide), which
 also defines the design, architecture and style for Element too.
 
 Before starting work on a feature, it's best to ensure your plan aligns well
-with our vision for Riot. Please chat with the team in
+with our vision for Element. Please chat with the team in
 [#riot-dev:matrix.org](https://matrix.to/#/#riot-dev:matrix.org) before you
```
Comment (jryans) [inline README.md:195]: The room alias _has_ changed to `#element-dev`, so likely good to update it.

File: README.md (hunk 1 above)
Comment (Weasy666) [reply README.md]: Ok, i reverted this. In favor of changing it after #14896 landed.

File: README.md (hunk 2 above)
Comment (Weasy666) [reply README.md]: Ok, i reverted this and will leave a comment on that issue.

File: README.md (hunk 3 above)
Comment (Weasy666) [reply README.md]: Done!

**Review by jryans — APPROVED** [review]
Thanks for all the revisions, looks great to me. 😄

**Comment by Weasy666** [comment]
Thanks for the patience 😅
And you are welcome.

**Pull request merged**